spec/lib/auxiliary/test_helper_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../../../lib/auxiliary/time_helper.rb'
+
+RSpec.describe AutoHCK::Helper do
+  include AutoHCK::Helper
+
+  describe '#time_to_seconds' do
+    
+    time_strings = {
+      "1:2" => 3720, #hh:mm
```

Comment:
I don't prefer to have magic numbers without comments. 

You wrote `"1.2:3:4" => 86400 + 7380 + 4`, then I preferred `"1.2:3:4" => 86400 + 2 * 60 * 60 + 3 * 60 + 4` 
Other variants add comments before `time_strings` that the number is a second count and wrote `93784` instead of  `86400 + 7380 + 4`

Currently, it looks strange why we sometimes calculate something but sometimes have a constant.